### PR TITLE
RELATED: ONE-3918 updated changelog

### PR DESCRIPTION
**Pull request by yanlipnican**
Updated changelog

---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4505/ : SUCCESS in 9m 25s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5776/ : SUCCESS in 10s

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,13 @@ The REST API versions in the table are just for your information as the values a
 
 ### Changed
 
+- Updated Highcharts to version `7.1.1`
+- Updated Ag-grid to community version `20.0.0`
+- Added carets `^` to versions of:
+    - React `^16.5.2`
+    - Lodash `^4.17.11`
+    - And other smaller third party libraries
```
Comment (evgeniabogranskaya) [inline CHANGELOG.md]: And other smaller third-party libraries

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2414/ : SUCCESS in 13m 43s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4507/ : SUCCESS in 10m 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1141/ : SUCCESS in 33s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1446/ : SUCCESS in 23m 13s

**Pull request merged**